### DISCUSSION TITLE
How do I update the software?

### DISCUSSION THREAD

**Discussion by Brazedd** [AlexandreRouma/SDRPlusPlus#1107] [Q&A]
The obvious answer is to delete the last files build and make a new one but id like to keep all my settings and saved frequency list. I've gone through the manual and can't seem to find the answer. It's not a common question from my Google searches so I'm missing something pretty obvious.

**Comment by AlexandreRouma** [comment]
On windows you just drop in the files on top of your existing install, on linux you just install with the package manager and it'll handle everything automatically.

**Reply by Brazedd** [reply]
Thank you for the help!

**Reply by Brazedd** [reply]
That did not work they are all gone now...I just dragged all the new files from the zip into the old folder and replaced its nuked everything

**Reply by AlexandreRouma** [reply]
What version were you updating from?

**Reply by Brazedd** [reply]
The last one. not sure what it was called but this was my first update

**Comment by s-petersen** [comment]
On Windows, it looks like you save the latest dated Json files in a different location, install the latest build, and copy back the json files overwriting the new install json files and it should save the config.
